<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="chatBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>401</width>
      <height>111</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="chatInputLE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="tileView">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>580</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>0.000000000000000</width>
      <height>0.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QPushButton" name="endTurnBtn">
    <property name="geometry">
     <rect>
      <x>1169</x>
      <y>690</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>End turn</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
